tests/cases/docker/test262/typescript.js

```diff
@@ -0,0 +1,5 @@
+const ts = require("typescript");
+
+module.exports = function(code) {
+    return ts.transpile(code);

```

Comment:
adding a try/catch here might prevent the early crashing. and some logging might give some insight in why the test262-harness is crashing.

Reply:
The thrown error includes the stack for the crash, but decorating the error with the input code and rethrowing it should make it trivial to extract a repro (or logging it and still executing other tests, but I hope these crashes are infrequent enough that we don't "expect" any crashes, so I'd like to avoid that).